### DISCUSSION TITLE
make new versin

### DISCUSSION THREAD

**Discussion by DeviateBacon3** [ThioJoe/YT-Spammer-Purge#108] [Ideas]
make one for android

**Comment by DeviateBacon3** [comment]
so i can get for my linux/chomebook

**Reply by TechStudent10** [reply]
If you have ChromeOS or any Linux distro, it comes included with Python. Just download the repo, install all the dependencies (via the terminal) and run. No need for a `.apk` or `.AppImage`.

**Comment by TechStudent10** [comment]
@DeviateBacon3 If you have ChromeOS or any Linux distro, it comes included with Python. Just download the repo, install all the dependencies (via the terminal) and run. No need for a `.apk` or `.AppImage`.

**Reply by KendallDoesCoding** [reply]
> @DeviateBacon3 If you have ChromeOS or any Linux distro, it comes included with Python. Just download the repo, install all the dependencies (via the terminal) and run. No need for a `.apk` or `.AppImage`.

Yeah!!

**Comment by KendallDoesCoding** [comment]
> make one for android

Your always welcome to run a windows/linux/mac emulator on your phone.

**Comment by ThioJoe** [comment]
There is a 0% chance I make it for android

**Reply by TechStudent10** [reply]
yep. can totally agree with that.

**Comment by KendallDoesCoding** [comment]
Why do we need one for android, anyway?

**Comment by KendallDoesCoding** [comment]
To make this program work on android, I got a pull request coming soon.

**Comment by KendallDoesCoding** [comment]
> make one for android

I got a solution for it to be able to work on android #118
